src/main/java/com/harrison/sort/BubbleSort.java

```diff
@@ -0,0 +1,11 @@
+package com.harrison.sort;
+
+public class BubbleSort {
+  
+    public String[] sort(String[] skills) {
+        String[] sorted = new String[skills.length];
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "sorted". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS1488)